apps/assisted_tagging/serializers.py

```diff
@@ -127,3 +127,51 @@ def validate(self, data):
                 association='Association is required for this widget.'
             ))
         return data
+
+
+class AutoDraftEntryGqlSerializer(ProjectPropertySerializerMixin, UserResourceCreatedMixin, serializers.ModelSerializer):
+    class Meta:
+        model = DraftEntry
+        fields = (
+            'lead',
+            'is_discarded'
+        )
+
+    def validate_lead(self, lead):
+        if lead.project != self.project:
+            raise serializers.ValidationError('Only lead from current project are allowed.')
+        af = lead.project.analysis_framework
+        if af is None or not af.assisted_tagging_enabled:
+            raise serializers.ValidationError('Assisted tagging is disabled for the Framework used by this project.')
+        return lead
+
+    def validate(self, data):
+        if self.instance and self.instance.created_by != self.context['request'].user:
+            raise serializers.ValidationError('Only reviewer can edit this review')
+        data['project'] = self.project
+        if self.project.is_private:
+            raise serializers.ValidationError('Assisted tagging is not available for private projects.')
+        return data
+
+    def create(self, data):
+        lead = Lead.objects.get(id=self.data['lead'])
+        if lead.auto_entry_extraction_status == (Lead.AutoExtractionStatus.SUCCESS):
+            raise serializers.DjangoValidationError("Already Tiggered")
+        if not lead.leadpreview.text_extract:
+            raise serializers.DjangoValidationError('Simplifed Text is empty')
+        draft_entry = DraftEntry.objects.filter(lead=self.data['lead'], draft_entry_type=0)
+        if draft_entry:
+            raise serializers.DjangoValidationError('Draft entry already exit')
+        # Use already existing draft entry if found
+        # Create new one and send trigger to deepl
+        lead.auto_entry_extraction_status = Lead.AutoExtractionStatus.PENDING
+        lead.save()
```

Comment:
Let's use update_fields when we are just changing one.

Reply:
Let's search for `save()` and if we are just changing one field. then let's use update_fields